test/model/calculations.test.ts

```diff
@@ -233,3 +233,110 @@ describe(Calculations.OptionsRemainingCalculator, () => {
     expect(subject.value).toBe(100);
   });
 });
+
+describe(Calculations.ConversionRatioCalculator, () => {
```

Comment:
All in all these tests are good! I'd like to see the setup code simplified, and perhaps making some of the class IDs carry meaning would clarify things what is happening, but this is a solid start.